code/datums/datacore.dm

```diff
@@ -95,6 +96,26 @@ GLOBAL_DATUM_INIT(data_core, /datum/datacore, new)
 			crimes |= crime
 			return
 
+/**
+ * Marks a crime as served for on the security record.
+ *
+ * Is used change the status of a single crime between "Served" and "Not Served" someone's security record.
+ * Arguments:
+ * * id - record id.
+ * * cDataId - id of already existing crime.
+ */
+/datum/datacore/proc/serveCrime(id, cDataId)
+	for(var/datum/data/record/R in security)
+		if(R.fields["id"] == id)
```

Comment:
```suggestion
		if(R.fields["id"] != id)
		    continue
```
saves some indentation. also, the same for line 112

Reply:
Good idea. I was just mimicking the code from the other procs but this is indeed cleaner